docs/Advanced/Refs.mdx

```diff
@@ -69,14 +69,31 @@ export interface CanvasRef {
   containerHeight?: number;
 
   /**
-   * Center the canvas to the viewport.
+   * Positions the canvas to the viewport.
    */
-  centerCanvas?: () => void;
+  positionCanvas?: (position: CanvasPosition, animated?: boolean) => void;
 
   /**
    * Fit the canvas to the viewport.
    */
-  fitCanvas?: () => void;
+  fitCanvas?: (animated?: boolean) => void;
+
+  /**
+   * Fit a node to the viewport.
+   */
+  fitToNode?: (nodeId: string, animated?: boolean) => void;
+
+  /**
+   * Scroll to X/Y
+   */
+  setScrollXY?: (xy: [number, number], animated?: boolean) => void;
+
+  observe: (el: HTMLDivElement) => void;
```

Comment:
Whats this?

Reply:
I just pasted `CanvasRef` into these docs, i'm not positive what that does tho maybe we remove it from docs https://github.com/reaviz/reaflow/blob/28047d52acce5154ffce58dd394b0798df9732f3/src/layout/useLayout.ts#L94-L95

Reply:
Oh ok... i just saw the addition and was wondering considering there was no comments.